src/components/Pensjonssimulering/utils.ts

```diff
@@ -1,18 +1,37 @@
 import {
   AxisLabelsFormatterContextObject,
   Axis,
+  Chart,
+  Options,
   Point,
+  PointClickEventObject,
+  Series,
   TooltipFormatterContextObject,
-  Options,
 } from 'highcharts'
 
 import { formatAsDecimal } from '@/utils/currency'
 
 import globalClassNames from './Pensjonssimulering.module.scss'
+
 export const MAX_UTTAKSALDER = 78
 export const COLUMN_WIDTH = 25
 export const TOOLTIP_YPOS = 35
 
+export const SERIE_NAME_INNTEKT = 'Inntekt (lønn m.m.)'
```

Comment:
Veldig mange av disse constene blir bare brukt i utils-fila og trenger derfor ikke eksporteres. Syns også det er litt rart at noen av de bare blir brukt i `Pensjonssimulering.tsx` men ikke er deklarert der. Ikke viktig å endre her siden det uansett vil endres på i en PR jeg har, men tenkte bare påpeke det

Reply:
Jeg samler constants i utils filene for å holde komponenten kort og rent. De kan da importeres samlet på toppen av filen. Vi kan evt. ha egne adskilte constants filer om du foretrekker det, men jeg synes at det blir uoversiktlig å ha dem både i components filer og i utils filer. Fint om du ikke flytter dem PRen som du nevner.

Enig at de ikke bør eksporteres hvis ikke det er behov.